app/views/admin/admin_portals/index.html.erb

```diff
@@ -1,3 +1,15 @@
+<div>
+  <div class="tag-card crayons-card p-4 m:p-6 m:pt-4 mb-4 flex flex-row relative">
+    <div class="mr-8">
+      <div class="crayons-field__description">Last deployed</div>
```

Comment:
`FOREM_BUILD_DATE` is the datetime when the container was created. When we start the container `RELEASE_FOOTPRINT` is the point in time that the container was started or deployed. I would change this to Container Build Date and make another section for Last Deployed that uses `RELEASE_FOOTPRINT` for the last time Forem was deployed.

Reply:
@jdoss thank you so much for this 😊 I'll use RELEASE_FOOTPRINT for Last deploy date. I don't think our users would need to know the Container Build Date so I'll leave that out. Thanks!